### ISSUE TITLE
Error Messages/Response Consistency

### ISSUE THREAD

**Issue by kreynoldsf5** [f5devcentral/f5-cli#4]
Successful commands return valid JSON:
```shell
bash-5.0# f5 bigip extension as3 install
{
    "message": "Extension component package 'as3' version '3.18.0' is already installed"
}
```
This is easy to deal with in a pipeline or when automating -- it can be piped, parsed, and verified.

Invalid commands lead to a stack trace:
```shell
bash-5.0# f5 bigip extension as3 install --version example_version
Traceback (most recent call last):
  File "/usr/local/bin/f5", line 8, in <module>
    sys.exit(cli())
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/click/decorators.py", line 64, in new_func
    return ctx.invoke(f, obj, *args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/f5cli/commands/cmd_bigip/__init__.py", line 69, in command_as3
    package_url)
  File "/usr/local/lib/python3.7/site-packages/f5cli/commands/cmd_bigip/extension_operations.py", line 105, in __init__
    self._extension_client = self._extension_client_attr(self._mgmt_client, **component_kwargs)
  File "/usr/local/lib/python3.7/site-packages/f5sdk/bigip/extension/extension_as3.py", line 49, in __init__
    use_latest_metadata=kwargs.pop('use_latest_metadata', False)
  File "/usr/local/lib/python3.7/site-packages/f5sdk/bigip/extension/extension_metadata.py", line 68, in __init__
    version or self.get_latest_version()
  File "/usr/local/lib/python3.7/site-packages/f5sdk/bigip/extension/extension_metadata.py", line 157, in _validate_component_version
    'Valid component version must be provided: %s' % (versions)
f5sdk.exceptions.InvalidComponentVersionError: Valid component version must be provided: ['3.18.0', '3.17.1', '3.17.0', '3.16.0', '3.15.1', '3.14.1', '3.15.0', '3.13.2', '3.14.0', '3.13.1', '3.13.0', '3.12.0', '3.11.0', '3.10.0', '3.9.0', '3.8.1', '3.8.0', '3.7.0', '3.5.1', '3.6.0', '3.5.0', '3.4.0', '3.3.0', '3.2.0', '3.1.0', '3.0.0']
```

The exception handler exists with very competent and useful error message -- but there isn't much that can done with the stack trace (hard to parse). The CLI should catch all exceptions and return them as valid JSON. Errors should still result in a non-zero exit code. 

This would make the user experience more enjoyable even if being used interactively. User's of the CLI don't need the stack trace by default -- they are likely not going to doing PRs against this project.

**Comment by shyawnkarim** [comment]
Thanks for reaching out to us with this request. Internal issue AUTOSDK-281 has been created for this enhancement.

**Comment by shyawnkarim** [comment]
Closing. Work on this enhancement has been completed and will be included in release 0.9.3.
